src/release.py

```diff
@@ -344,6 +345,7 @@ def run():
         resp = fetch_history_from_github(github_user, github_project, first_commit)
 
     commits_since_tag = []
+    bug_name_prefix_regexp = re.compile(re.escape('bug-'), re.IGNORECASE)
```

Comment:
I'd restrict the pattern further so it matches `bug-[\d]+`. Otherwise it's going to match `debug-sym-file.py` and other things we have.